Makefile

```diff
@@ -2,13 +2,20 @@ SHELL                   = /bin/bash
 
 ROS_DISTRO             ?= noetic
 BUILDER_NAME            = seqsense/aports-ros-builder
-ALPINE_VERSION          = $(shell . ./alpine_version_from_ros_distro.sh; alpine_version_from_ros_distro $(ROS_DISTRO))
+ALPINE_VERSION          = 3.11
```

Comment:
Should it be `ALPINE_VERSION          ?= 3.11` so that `ALPINE_VERSION` can be set to `3.14`?

Reply:
`ALPINE_VERSION = 3.11` can be overwritten by `make ALPINE_VERSION=3.14` as well.
But `?=` may be better for readability to mark it is configurable.